src/toil/common.py

```diff
@@ -181,6 +181,12 @@ def parse_int_list(s: str):
             self.workDir = os.path.abspath(self.workDir)
             if not os.path.exists(self.workDir):
                 raise RuntimeError(f"The path provided to --workDir ({self.workDir}) does not exist.")
+
+            if len(self.workDir) > 80:
+                logger.warning(f'Length of workDir path "{self.workDir}" is {len(self.workDir)} char long.  '
```

Comment:
`Length of XXX is XXXX char long` is a construction that upsets me. Make it `Length of workDir path "{self.workDir}" is {len(self.workDir)} characters.` instead.